frontend/assets/stylesheets/components/_action.scss

```diff
@@ -97,6 +97,29 @@
     cursor: default;
 }
 
+.action--snap {
```

Comment:
This is a little awkward but feels OK for now. This is to create an action that starts as circle/arrow only and then shows the label on larger screens. Naming things is hard.